components/x-privacy-manager/storybook/data.js

```diff
@@ -1,26 +1,59 @@
 const CONSENT_API = 'https://consent.ft.com'
 
-const defaults = {
+const referrers = {
+	'ft.com': 'www.ft.com',
+	'exec-appointments.com': 'www.exec-appointments.com',
+	'fdibenchmark.com': 'www.fdibenchmark.com',
+	'fdiintelligence.com': 'www.fdiintelligence.com',
+	'fdimarkets.com': 'www.fdimarkets.com',
+	'fdireports.com': 'www.fdireports.com',
+	'ftadviser.com': 'www.ftadviser.com',
+	'ftconfidentialresearch.com': 'www.ftconfidentialresearch.com',
+	'globalriskregulator.com': 'www.globalriskregulator.com',
+	'investorschronicle.co.uk': 'www.investorschronicle.co.uk',
+	'non-execs.com': 'www.non-execs.com',
+	'pensions-expert.com': 'www.pensions-expert.com',
+	'pwmnet.com': 'www.pwmnet.com',
+	'thebanker.com': 'www.thebanker.com',
+	'thebankerdatabase.com': 'www.thebankerdatabase.com',
+	Undefined: ''
```

Comment:
Is this needed?

Reply:
I am not sure why it was added in the first place, I tried making the referrer be `undefined` or `'Undefined'` to see if it would grab it as the selected option in the referrer's dropdown and it did not, I am removing it

Reply:
This is to allow the Storybook demonstrate behaviour for both ft.com and Specialist Title sites

Reply:
@oliverturner thanks! I'll rename it to `Default`